internal/hcl/parser.go

```diff
@@ -602,14 +628,3 @@ func (p *projectLocator) walkPaths(fullPath string, level int) []string {
 
 	return dirs
 }
-
-func (p *projectLocator) isSkipped(info os.DirEntry) bool {
-	for _, isSkipped := range p.excludedDirs {
-		if isSkipped(info.Name()) {
-			log.Debugf("skipping directory %s as it is marked as exluded by --exclude-path", info.Name())
```

Comment:
Do you want to keep this debug logging on line 536?

Reply:
👍